src/vcpkg-test/files.cpp

```diff
@@ -35,7 +35,10 @@ namespace
 
     bool is_valid_symlink_failure(const std::error_code& ec) noexcept
     {
-#if defined(_WIN32)
+#if defined(__MINGW32__)
+        // mingw doesn't support symlink operations
```

Comment:
It would be better to add a check for whatever error_code mingw returns in that case rather than just always saying true. They probably say something like `std::errc::not_supported` ?